go.mod

```diff
@@ -3,17 +3,18 @@ module github.com/nordix/meridio
 go 1.22
 
 require (
-	github.com/edwarnicke/grpcfd v1.1.2
+	github.com/edwarnicke/grpcfd v1.1.4
 	github.com/faisal-memon/sviddisk v0.0.0-20211007205134-77ccea0b9271
 	github.com/go-logr/logr v1.4.1
 	github.com/go-logr/zapr v1.3.0
 	github.com/golang/mock v1.6.0
+	github.com/golang/protobuf v1.5.4
```

Comment:
Could you do `go mod tidy`, I don't think this dependency should be a direct one.

Reply:
> Could you do `go mod tidy`, I don't think this dependency should be a direct one.

I did use go mod tidy the first place.